torch/csrc/jit/python/python_sugared_value.cpp

```diff
@@ -650,6 +650,7 @@ std::shared_ptr<SugaredValue> ModuleValue::call(
   // call pre_hooks
   if (have_pre_hooks) {
     for (const auto& hook : class_type->getForwardPreHooks()) {
+      assert(forward_input != nullptr);
```

Comment:
Consider using `TORCH_INTERNAL_ASSERT` to provide more information about this if it is false.

Reply:
I'm also curious why you added it.

Reply:
The internal FB linter asked for me to add it-- I think it's not necessary because it shouldn't be null with the logic above it. Are we allowed to ignore FB lint warnings?

Reply:
I'd prefer not to ignore it but I do think we should try to use `TORCH_INTERNAL_ASSERT` so that we can provide some context or information if the assert ever trips.